cpp/oneapi/dal/algo/decision_forest/backend/gpu/train_kernel_regression_hist_dpc.cpp

```diff
@@ -0,0 +1,185 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#define DAAL_SYCL_INTERFACE
+#define DAAL_SYCL_INTERFACE_USM
+#define DAAL_SYCL_INTERFACE_REVERSED_RANGE
+
+#include <daal/include/services/error_handling.h>
```

Comment:
Do you really need this include?

Reply:
Yes, i need #define DAAL_SYCL_INTERFACE_REVERSED_RANGE to have correct order in ranges assignment

Reply:
regarding include -> removed it, thanks